File: java/Java入門/problem_019.java
// 数の表示とサイコロ（演習２）

public class Main {
	public static void main(String[] args) {
		double rand = Math.random() * 50 + 50;     //Math.random() は0.0～1.0の範囲で乱数を生成するため、範囲を (99 - 50 + 1) つまり 50 とし、これに50を足すことで50～99の範囲のランダムな値を生成しています。
		int number = (int)rand;
		System.out.println("モンスターに、" + number + "のダメージを与えた");
	}
}

File: Java Script/Java Script入門/problem_019.js
//値段を計算する
process.stdin.resume();
process.stdin.setEncoding('utf8');
var apple_price = (parseInt(Math.random() * 3) + 1) * 100;   //リンゴの単価,リンゴの単価を100,200,300とランダムに選択((parseIntを()で囲んで最後に* 100をする)
var apple_num = parseInt(Math.random() * 10) + 1;    //リンゴを買う数,random関数に書き換える（ランダムに選択される数を〜10に）
console.log("リンゴの単価 : " + apple_price + "円" );
console.log("リンゴの個数 : " + apple_num + "個");
var total = apple_price * apple_num;       //変数(total)にapple_priceとapple_numをかけた金額を表示
console.log("合計金額 : " + total + "円");